docs/source/autograd.rst

```diff
@@ -43,27 +30,44 @@ actually lower memory usage by any significant amount. Unless you're operating
 under heavy memory pressure, you might never need to use them.
 
 In-place correctness checks
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------------------
 
-All :class:`Variable` s keep track of in-place operations applied to them, and
-if the implementation detects that a variable was saved for backward in one of
+All :class:`Tensor` s keep track of in-place operations applied to them, and
+if the implementation detects that a tensor was saved for backward in one of
 the functions, but it was modified in-place afterwards, an error will be raised
 once backward pass is started. This ensures that if you're using in-place
 functions and not seeing any errors, you can be sure that the computed
 gradients are correct.
 
+Variable (deprecated)
+^^^^^^^^^^^^^^^^^^^^^
 
-.. autoclass:: Variable
-    :members:
+.. warning::
+    The Variable API has been deprecated: Variables are no longer necessary to
+    use autograd with tensors. Autograd automatically supports Tensors with
+    ``requires_grad`` set to ``True``. In addition, ``Variable(tensor)`` now returns
+    a :class:`torch.Tensor`. Below please find a quick guide on what has changed:
+
+    - ``Variable(tensor, requires_grad=True)`` is now ``torch.tensor(tensor, requires_grad=True)``
+    - ``var.data`` is now just ``tensor``. If you don't want to record operations on tensors,
```

Comment:
Could you also a line saying that one can set `requires_grad=True` with tensor factory methods, e.g., `torch.randn(3, 3, requires_grad=True)`?